src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -84,6 +84,8 @@ def load_arguments(self, _):
                        help='Runtime version of used language')
             c.argument('jvm_options', type=str,
                        help="A string containing jvm options, use '=' instead of ' ' for this argument to avoid bash parse error, eg: --jvm-options='-Xms1024m -Xmx2048m'")
+            c.argument('net_core_main_entry_path', type=str,
+                       help="A string containing the path to the .NET executable relative to zip root")
             c.argument('env', env_type)
 
     for scope in ['spring-cloud app create', 'spring-cloud app deployment create']:
```

Comment:
Do we need to rename (or maybe add new argument for backward compatibility) the `jar_path` argument (https://github.com/Azure/azure-cli-extensions/pull/1371/files#diff-31fe45cdcec6a27226e759744f994e10R112) here? For example, `artifact_path` / `binary_path`, since we will cover both `JAR` and `ZIP` here.

Reply:
From UX perspective i'm ok to deprecate --jar-path and use --artifact-path instead.
@ShichaoQiu any input from eng point of view? will supporting both .NET and java artifact with one parameter cause any problems?